### PR TITLE
Get flag summary from latest msutils-1.2.0 with plots

### LINKED ISSUE #1292: flag worker includes worker name in output products twice
Our current flagging worker likes likes to say things things twice:
```
logs-20201126-232908/log-flag-flag__target-spw-ms1-20201126-232908.txt
logs-20201126-232908/log-flag-flag__target-mask-ms1-20201126-232908.txt
logs-20201126-232908/log-flag-flag__target-manual-ms1-1-20201126-232908.txt
logs-20201126-232908/log-flag-flag__target-rfi-ms1-20201126-232908.txt
logs-20201126-232908/log-flag-flag__target-summary-ms1-20201126-232908.txt
logs-20201126-232908/log-flag-flag__target-summary-ms1-20201126-232908.json
```
The first "flag" comes from the label passed to ``Stimela.Recipe()`` when this is created at worker administrator level. At @francescaLoi 's behest I'm changing this to the worker name rather than worker type, because that just makes more sense.

The second "flag__xxx" comes from the worker name being explicitly inserted into the ``name`` argument of each ``recipe.add`` within the flag worker itself. No other worker follows this convention. There is no harm in this, but it's a bit annoying to sometimes see things twice twice but not quite always. (Obligatory [Python Python reference](https://www.youtube.com/watch?v=IdTy1j7z-K4).)

I propose to (1) change the worker admin to insert worker name rather than type (so, ``log-flag__target-xxx``), (2) fix the flag worker to not insert the worker name into its step names.

This, however, turns up this bit of [code from the naughty corner](https://github.com/caracal-pipeline/caracal/blame/master/caracal/workers/flag_worker.py#L523). Tut tut, relying on output filenames to be named in a certain specific precise way, when this is not documented and not under our control (what if Stimela changes its log naming convention slightly?) I propose to change this to get the log filename via an API call.

### PR DESCRIPTION
**Pull request by Athanaseus**
- Flags stats from `msutils`
- Remove bokeh
- resolves #1292

### CODE REVIEW

**Comment by o-smirnov** [comment]
ok to test

**Review by o-smirnov — APPROVED** [review]
LGTM. I assume you've checked that the plots are generated as before?

**Comment by Athanaseus** [comment]
yes, I tested that.

File: caracal/workers/selfcal_worker.py [hunk 1]
```diff
@@ -2329,10 +2329,10 @@ def ragavi_plotting_cubical_tables():
                                "plot": True,
                                "outfile": ('{0:s}-{1:s}-'
                                            'slefcal-summary-{2:d}.json').format(
```
[the remark this answers is not in the document]
Comment (o-smirnov) [reply caracal/workers/selfcal_worker.py:2331]: @Athanaseus could you please fix the typo in the filename.

**Review by o-smirnov — CHANGES_REQUESTED** [review]

**Review by o-smirnov — APPROVED** [review]

**Pull request merged**